Tools/Scripts/webkitperl/test262_unittest/fixtures/test/test-that-passes/pass.js

```diff
@@ -0,0 +1,9 @@
+// Copyright (C) 2018 Valerie Young. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-function-definitions
+description: Minimal test
+---*/
+
+function f() {};
+
```

Comment:
Potentially clearer naming and directory conventions:

Option one - skip folders since these are only fixtures.

```
── fixtures
    ├── test 
    │   ├──expected-to-pass.js
    │   ├──expected-to-fail.js
    │   ├── expected-to-pass-now-failing.js
    │   ├── expected-to-fail-now-passing.js
    │   ├── expected-to-fail-now-failing-with-new-error.js
```
Feel free to look at the WPT Webkit scenarios matrix @bmac and I worked on for inspiration on more test scenarios. https://github.com/bocoup/webkit/wiki/WPT-Import-Test-Scenario-Matrix 

Renaming the test directory to `test-scenarios` might be nice too.